app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils desktop
+
+DESCRIPTION="A complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+COMMON_DEPEND="
+	>=media-gfx/sane-backends-1.0
+	>=dev-qt/qtsvg-5.4.1:5
```

Comment:
Please keep deps sorted lexicographically.
Also, I'm quite sure you'll also need to depend on `qtcore` here.

Reply:
Given that qtsvg depends on qtcore is it really necessary? Just wondering.

Reply:
yes, we don't rely on implicit deps.

Reply:
OK, just performed the required changes. Thanks.